specifications.yml

```diff
@@ -1007,6 +1109,123 @@ components:
+        creditInsuranceLimitCurrency:
+          type: string
+          description: "Currency for the insurance credit limit"
+
+    termsForFinancingApplication:
+      type: object
+      description: Terms of the application. (Look at the examples on the right to understand when they are required)
+      properties:
+        financeAmount:
+          type: number
+          description: Amount to be financed
+        feeSettings:
+          type: array
+          description: "List of settings for the fee"
+          items:
+            $ref: "#/components/schemas/feeSetting"
+    
+    feeSetting:
+      type: object
+      properties: 
+        rateType:
+          $ref: "#/components/schemas/rateType"
+        feeTrigger:
+          $ref: "#/components/schemas/feeTrigger"
+        feeReferenceAmount:
+          $ref: "#/components/schemas/feeReferenceAmount"
+        dayCountConvention:
+          $ref: "#/components/schemas/dayCountConvention"
+        percentageValue:
+          type: number
+          description: "Percentage of the fee"
+        billingMethod:
+          $ref: "#/components/schemas/billingMethod"
+
+    rateType:
+      type: string
+      enum:
+        - flat
+        - variable
+      description: |
+        Rate type of a fee.
+        Possible values are
+        * flat -> the rate of the fee remains constant.
+        * variable -> the rate of the fee can change.
+    
+    feeTrigger:
+      type: string
+      enum: 
+        - 'monthly-until-paid'
+        - 'on-financing'
+        - 'on-payment'
+      description: |
+        When the fee is calculated.
+        Possible values are
+        * monthly-until-paid -> the fee is calculated monthly until the invoice is paid.
+        * on-financing -> the fee is calculated at the moment of financing.
+        * on-payment -> the fee is calculate at the moment the invoice is paid.
+    
+    feeReferenceAmount:
+      type: string
+      enum:
+        - 'invoice-amount'
+        - 'financed-amount'
+      description: |
+        The reference used to calculate the fee.
+        Possible values are
+        * invoice-amount -> the fee is calculated on the invoice amount.
+        * financed-amount -> the fee is calculated on the finance amount.
+    
+    dayCountConvention:
+      type: string
+      enum:
+        - 'actual-360'
+        - 'actual-365'
+      description: |
+        Number of day on which the interest is calculated.
+        Possible values are
+        * actual-360 -> the fee is calculated on 360 days.
+        * actual-365 -> the fee is calculated on 365 days.
+
+    billingMethod:
+      type: string
+      enum:
+        - 'deduct'
+        - 'separate'
+      description: |
+        Methodology used for billing.
+        Possible values are
+        * deduct -> The value of the fee is included in the financed amount.
+        * separate -> The value of the fee is not included in the financed amount.
```

Comment:
Here it is also important that we mention that the separate fees will be debited in bulk at the end of the month